packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Prop,
+  State,
+} from '@stencil/core';
+import { getSlottedNodes } from '../../utils/utils';
+
+@Component({
+  tag: 'va-breadcrumbs',
+  styleUrl: 'va-breadcrumbs.css',
+  shadow: true,
+})
+export class VaBreadcrumbs {
+  @Element() el: HTMLElement;
+
+  /**
+   * Adds an aria-label attribute to the <nav /> element.
+   */
+  @Prop() label: string = 'Breadcrumb';
+
+  /**
+   * Analytics tracking function(s) will not be called
+   */
+  @Prop() disableAnalytics: boolean;
+
+  /**
+   * Adds a custom id attribute to the UL element
+   */
+  @Prop() listId: string;
```

Comment:
I'm not sure if we need this. I know it's on the current `<Breadcrumbs>` react component, but it doesn't appear to be used.

![image](https://user-images.githubusercontent.com/2008881/145624653-13178c44-4a36-4303-9f0f-b406662f18b5.png)

Also, since the `<ul>` for this component is inside the shadow DOM, I don't think that certain things like `aria-describedby` would be able to successfully point to a custom ID inside of the component.